lib/recurly/account.rb

```diff
@@ -42,6 +42,9 @@ class Account < Resource
     # @return [Pager<CreditPayment>, []]
     has_many :credit_payments, class_name: :CreditPayment, readonly: true
 
+    # @return [Pager<CustomField>, []]
```

Comment:
Sorry, one more small detail. Let's change this since it will never be wrapped in a pager. Should be `[[CustomField], []]`